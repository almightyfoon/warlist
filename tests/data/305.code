c9501b_-0k6I4x4C5D6v6vgg5U6J6p6F6p6F6WhV__444xcQcvcv7y6yfs5U6767