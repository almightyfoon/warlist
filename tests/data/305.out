https://conflictchamber.com/#c9501b_-0k6I4x4Cn66v6vgg5U6J6p6F6p6F6WhV__444xn0cvcv7y6yfs5U6767

Skorne Army - 74 / 75 points
[Theme] Winds of Death

[Rasheth 1] Dominar Rasheth [+28]
 - Basilisk Krea [7]
 - Cyclops Shaman [8]
 - Titan Cannoneer [16]
Extoller Soulward [0(3)]
Extoller Soulward [3]
Venator Dakar [0(4)]
Paingiver Beast Handlers (min) [5]
Venator Flayer Cannon Crew [0(5)]
Venator Reivers (max) [15]
 - Venator Reiver Officer & Standard [4]
Venator Reivers (max) [15]
 - Venator Reiver Officer & Standard [4]
Venator Slingers (min) [8]
Siege Animantarax [17]

Skorne Army - 75 / 75 points

[Makeda 2] Supreme Archdomina Makeda [+27]
 - Basilisk Krea [7]
 - Despoiler [17]
Mortitheurge Willbreaker [4]
Mortitheurge Willbreaker [4]
Orin Midwinter, Rogue Inquisitor [5]
Tyrant Rhadeim [9]
Legends of Halaak [8]
Paingiver Beast Handlers (min) [5]
Praetorian Ferox (max) [20]
Praetorian Ferox (max) [20]
